framework/gui.cpp

```diff
@@ -106,6 +106,7 @@ Gui::Gui(VulkanSample &sample_, const Window &window, const Stats *stats,
 	ImGuiStyle &style = ImGui::GetStyle();
 
 	// Color scheme
+	style.Colors[ImGuiCol_WindowBg]         = ImVec4(0.005f, 0.005f, 0.005f, 0.94f);
```

Comment:
Minor suggestion: We should also adjust the border color, as the default one results in a bright, almost white, very noticeable border.

This should look good and close to how it was before:

```cpp
style.Colors[ImGuiCol_Border] = ImVec4(0.02f, 0.02f, 0.02f, 0.94f);
```

Reply:
Right. I somehow hat borders enabled for a local branch. Forgot we disabled them for this repo.

Reply:
Ok - that explains it